api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -26,8 +26,7 @@ message GcpAuthnFilterConfig {
   // The URL format is "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=[AUDIENCE]"
   config.core.v3.HttpUri http_uri = 1 [(validate.rules).message = {required: true}];
 
-  // Retry policy for fetching tokens.
-  // This field is optional. If it is not configured, the filter will be fail-closed (i.e., reject the requests).
+  // Retry policy for fetching tokens. This field is optional.
```

Comment:
Can we document somewhere that this actually fail open? Or can we have it fail close? Given this is an alpha extension this behavioral change should be fine. IMO the latter makes more sense as it doesn't tie up resources unnecessarily.

Reply:
fine having the failure mode policy be some future work.